lib/utils/ConfigParser.js

```diff
@@ -272,6 +258,42 @@ class ConfigParser {
         return this._capabilities
     }
 
+    /**
+     * sets config attribute with file paths from filtering
+     * options from cli argument
+     *
+     * @param  {String} filterObj  list of files in a string from
+     * cli argument
+     * @return {Boolean} exclude  true or false
+     */
+    setFilePathToFilterOptions (filterObj, config) {
+        const objs = new Set()
+        const objList = filterObj.split(/,/g)
```

Comment:
instead of `objList` and `objsList` could we give more meaningful names?

Reply:
changed, hopefully it will make more sense now.